source/2019-09-20-ember-3-13-released.md

```diff
@@ -1,34 +1,46 @@
+Today, the Ember project is releasing version 3.13 of Ember.js, Ember Data, and Ember CLI.
 
-Additionally, versions 3.12 of Ember and Ember Data are now promoted to LTS, which stands for Long Term Support. An LTS version of Ember continues to receive security updates for 9 release cycles (54 weeks) and bugfixes for 6 cycles (36 weeks). LTS releases typically occur every four minor versions. The previous LTS versions for Ember and Ember Data were 3.4.
+Ember 3.13 is the final release of Ember before Octane. Ember 3.13 is also the Octane preview. The preview allows adventurous community members to help us put the final round of polish on the edition before Octane becomes official in 3.14.
 
-If you are upgrading from the previous LTS version to 3.12 then you are getting access to these features:
+Additionally, version 3.12 of Ember is now promoted to LTS (Long Term Support). An LTS version of Ember continues to receive security updates for 9 release cycles (54 weeks) and bugfixes for 6 cycles (36 weeks). LTS releases typically occur every four minor versions. The previous LTS version of Ember was 3.8.
 
-* `router` service 
-* Support for native classes
-* The array helper
-* Element Modifier Manager
-* Nested angle bracket component invocation
-* Angle bracket invocation for built-in components
-* Support for native decorators
-* Forwarding Element Modifiers with `...attributes`
-* The `{{fn}}` and `{{on}}` modifiers
+## The Octane Preview
 
-This release kicks off the 3.14 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+The Octane features in 3.13 are stable public APIs with our normal SemVer guarantees. If you build an app using APIs in the Octane preview, it will continue to work in future versions.
 
-You can read more about our general release process here:
+So why is this a preview and not the final Octane release?
 
-- [Release Dashboard](http://emberjs.com/builds/)
-- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
-- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
-- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+While these APIs are stable, not all Octane features are compatible with popular addons yet. We are also still updating documentation, and important tools like the Ember Inspector do not yet work with some Octane features.
+
+In Ember 3.14, Octane will be the recommended way to use Ember and new apps will have Octane's optional features enabled by default. The guides and tutorials will show Octane examples, and codemods will be available to help users migrate to Octane.
```

Comment:
```suggestion
In Ember 3.14, Octane will be the recommended way to use Ember and new apps will have Octane's optional features enabled by default. The [guides](https://octane-guides-preview.emberjs.com/release/) and [tutorial](https://octane-guides-preview.emberjs.com/release/tutorial/00-part-1/) will show Octane examples, and codemods will be available to help users migrate to Octane.
```

Reply:
It may be a good idea to pull this out higher up and explicitly call out the Octane Guides Preview, and in particular, the Tutorial is a great way for existing Ember users to get up to speed on Octane. I think the tutorial is worth calling out because I wrote it and am biased 😉, but also the other sections of the guides are in various state of WIP as we get ready for the 3.14 release, but I'm pretty confident that the tutorial is up-to-date and ready for use (and feedback, of course).